tools/experiments/idle_ext_pmt.py

```diff
@@ -0,0 +1,291 @@
+import pickle as _p
+import time as _t
+
+import numpy as _np
+from artiq.experiment import *
+from jax import InfiniteLoop, SinaraEnvironment
+
+__all__ = ["IDLE_EXT_PMT"]
+
+
+class IDLE_EXT_PMT(InfiniteLoop, SinaraEnvironment):
+    """Base class for background experiment to read PMT and set TTL/DDS parameters from an Arduino.
+
+    Inherit this class in the experiment repository and define the class variables:
+        REPUMP_AOM_CHANNELS: list of strs, names of DDSes controlling repump lasers.
+            To control ions that have hyperfine structures, multiple repump DDSes may be necessary.
+        PMT_EDGECOUNTER: str, edgecounter device for PMT input.
+
+    This experiment assumes that the device has at least one AD9910 DDS and at least one TTL board.
+    """
+
+    DIFFERENTIAL_TRIGGER = None
+    REPUMP_AOM_CHANNELS = None
+    kernel_invariants = {
+        "DIFFERENTIAL_TRIGGER",
+        "REPUMP_AOM_CHANNELS",
+        "PMT_EDGECOUNTER",
+        "repump_aoms",
+        "pmt_counter",
```

Comment:
Delete as it is no longer used.

Reply:
just removed pmt_counter and PMT_EDGECOUNTER